src/Interface/Lists/Radarcord.ts

```diff
@@ -5,30 +5,30 @@ import { Util, IDResolvable } from '../../Utils/Util'
  * Represents the Radar Bot Directory service.
  * @see https://docs.radarbotdirectory.xyz/
  */
-export default class RadarBotDirectory extends Service {
+export default class Radarcord extends Service {
   /** The values that can be used to select the service. */
   static get aliases() {
-    return ['radar', 'radarbots', 'radarbotdirectory.xyz']
+    return ['radar', 'radarcord', 'radarcord.net']
```

Comment:
```suggestion
    return ['radar', 'radarbots', 'radarbotdirectory.xyz', 'radarcord', 'radarcord.net']
```

Let's keep the old ones for backwards compatibility